### ISSUE TITLE
Package Ethic

### ISSUE THREAD

**Issue by feddelegrand7** [the-y-company/many#6]
@JohnCoene if you don't have anything to hide. Why did you delete my issue about the package that I created months ago and does exactly the same thing. I just asked that you cite the job that's already available.

**Comment by feddelegrand7** [comment]
I mean I don't really care if you use my idea without mentioning it but at least I was expecting some ethic from you. Deleting issues is low. Sad to see that.

**Comment by ppssphysics** [comment]
Dear @feddelegrand7, in the name of The Y Company, I apologize for what is happening here and the frustration it has caused you. I understand the situation. It is fair that the documentation of the many package we have recently released refers to your work.

The fact your package was published on CRAN before the many package was developed is undeniable, and we will recognize that. Please give us some time to update the documentation accordingly. We will also do our best to bring back the deleted issue. Deleting the issue was not an appropriate action.

With that said, your accusations of plagiarism on social media are extremely serious. We have conducted a code review of your package and the many package, and we have not found any evidence of code plagiarism. If you do, please share it with us, and we will take the appropriate measures.

Regarding getting inspiration from your initial idea, it is simply wrong. It is true we ommitted checking if a similar package already existed. We will acknowledge the precedence of your work as it deserves. But we will not recognise something that is simply false.

Attacking the integrity of a talented developer, with the intention to shame him in front of the community is something we take very seriously. We hope it can be brought to end very quickly.

**Comment by feddelegrand7** [comment]
@ppssphysics thank your for your reply. I find it constructive and informative. I just want to point out that I tried to notify to the developer that a similar package existed. I was respectful and I didn't mean any harm. Kindly enough, the developer wrote a section about the package called (similar works) however without any justification or reason, the developer John Cohen decided to delete the section referring to my package and with all dishonesty decided to delete the issue that I created (issue Number 5) and this is what let me to speak about it on social media. I saw it as an injustice which I had to talk about. Concerning the accusations of plagiarism, I do believe you when you say that you didn't have any knowledge of my package however ignoring purposely the existence of a software that introduced the idea before is a form of plagiarism. I didn't ask for much, just a reference however what I get is a strong refusal and accusations from the author.

**Comment by feddelegrand7** [comment]
I need also to add that I didn't attack the integrity of the developer. I only spoke about facts which are debatable however in one of his tweet, the developer used an extremely vulgar word in front of the community and this is something that I do take very seriously.

**Comment by ppssphysics** [comment]
As you can now acknowledge from my reply, there is no intention to ignore the existence of your work purposely. There has never been such intention at any time. And since you know the work of the developer, then you have witnessed on multiple occasions how he recognizes the work of others. I was a published researcher like you. Problems of plagiarism are close to my heart.

I understand that specific actions triggered your decision to go public. But please consider that there can be other explanations for what happened than the malicious intention not to recognize your work. I sincerely believe there was a big misunderstanding in what happened.

I wish you the best. And yes, the rfold idea is a great one.

**Comment by feddelegrand7** [comment]
@ppssphysics I don't know you personally but I want to thank you for the professional words. As a sign of gratitude. I'll delete all my tweets about the developer or the {any} package and apologize publicly for the plagiarism accusations. (The post on LinkedIn was removed immediately after I created it)

**Comment by ppssphysics** [comment]
@feddelegrand7 This is much appreciated. Thank you. We will close this issue with the deployment of the updated documentation that recognizes the {rfold} package.

**Comment by feddelegrand7** [comment]
Thank you very much @ppssphysics

**Comment by feddelegrand7** [comment]
One last think @ppssphysics I have deleted my Tweets and would appreciate if the developer could do the same. Thanks in advance.

**Comment by ppssphysics** [comment]
Yes, it has been done. And the package read me had been updated as well with [fe9d6a3](https://github.com/the-y-company/many/commit/fe9d6a372c99ea410322092b78ae11a87cd08953).
